docs/13-system-tests.md

```diff
@@ -115,6 +115,9 @@ You can run a specific test suite like this:
 
 This can also be used with NightlyFunctionalSystemTestSuite.
 
+Note that some system tests are designed to run on at least 3 subnets in different availability zones. There is no
+guarantee that these tests will pass if you use less subnets than this.
```

Comment:
It would be worth clarifying why they might fail if fewer subnets are used - is it just because there might not be enough Fargate capacity available?